i18n/data/fr-FR.json

```diff
@@ -15,10 +15,10 @@
   "UIKit.CreatableSelectInput.noOptionsMessageWithInputValue": "Aucune option",
   "UIKit.CreatableSelectInput.noOptionsMessageWithoutInputValue": "Aucune option",
   "UIKit.CreatableSelectInput.placeholder": "Sélectionner...",
-  "UIKit.DateInput.placeholder": "MM/JJ/AAAA",
-  "UIKit.DateRangeInput.placeholder": "MM/JJ/AAAA - MM/JJ/AAAA",
-  "UIKit.DateTimeInput.placeholder": "MM/JJ/AAAA - HH:mm AM/PM",
```

Comment:
french doesn't use am/pm, and when the "value" is displayed and formatted with moment, it's shown in 24 hour time.